cobalt/browser/h5vcc_system/h5vcc_system_impl.cc

```diff
@@ -126,10 +139,34 @@ H5vccSystemImpl::H5vccSystemImpl(
     : content::DocumentService<mojom::H5vccSystem>(render_frame_host,
                                                    std::move(receiver)) {
   DETACH_FROM_THREAD(thread_checker_);
+#if BUILDFLAG(IS_STARBOARD)
+  const StarboardExtensionIfaApi* ifa_api =
+      static_cast<const StarboardExtensionIfaApi*>(
+          SbSystemGetExtension(kStarboardExtensionIfaName));
+  const bool is_ifa_version_supported = ifa_api && ifa_api->version >= 2;
+  if (is_ifa_version_supported) {
```

Comment:
Question: Is this `ifa_api` changing during the browser lifetime?
If not, could we just keep it in a member variable, and assume that
the version is the appropriate one if it's not null?